class NotFoundError(Exception):
    pass


class ValidationError(Exception):
    pass


class AuthError(Exception):
    pass
